ChildProject/projects.py

```diff
@@ -612,6 +612,15 @@ def get_recordings_from_list(
             _recordings = _recordings[
                 _recordings["recording_filename"].isin(recordings)
             ]
+            
+            if _recordings.shape[0] < len(recordings):
+                recs = pd.Series(recordings)
+                missing_recs = recs[~recs.isin(self.recordings['recording_filename'])].tolist()
+                #self.recordings[~self.recordings['recording_filename'].isin(recordings)]['recording_filename'].tolist()
+                raise ValueError("recordings {} were not found in the dataset index.\
+                                 Check the names and make sure they exist in\
+                                 'metadata/recordings.csv'".format(missing_recs))
+                
```

Comment:
That is unrelated to the new metrics pipeline, right?

Reply:
Yes, it is related to the `--reordings` option that appears in numerous pipelines. If the recordings given did not exist in the dataset, the command would just continue and not print anything, so raising an error semmed like a better way of handling things